code/game/objects/explosion_recursive.dm

```diff
@@ -71,7 +71,6 @@ var/explosion_in_progress = 0
 
 /turf/simulated/mineral
 	explosion_resistance = 2
-	outdoors = TRUE // for weather
```

Comment:
Why?

Reply:
This made weather effects show over rocks. They should of made the lavaland version of mineral/wall be outside instead of touching the root turf